src/main/java/pl/touk/sputnik/processor/checkstyle/CheckstyleProcessor.java

```diff
@@ -59,7 +62,16 @@ private Checker createChecker(@NotNull AuditListener auditListener) {
             Checker checker = new Checker();
             ClassLoader moduleClassLoader = Checker.class.getClassLoader();
             String configurationFile = getConfigurationFilename();
-            Properties properties = System.getProperties();// loadProperties(new File(System.getProperty(CHECKSTYLE_PROPERTIES_FILE)));
+            if (StringUtils.isBlank(configurationFile)) {
+                throw new ReviewException("Checkstyle configuration file is not specified.");
+            } else if (!Files.exists(Paths.get(configurationFile))) {
```

Comment:
You don't need else if you throw exception.

Reply:
done